src/PageLayout/Notification/Recurring/RecurringDeliveryRequestController.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+import '../Popup.css';
+
+import RecurringDeliveryRequest1 from './RecurringDeliveryRequest1';
+import RecurringDeliveryRequest2 from './RecurringDeliveryRequest2';
+import RecurringDeliveryRequest3 from './RecurringDeliveryRequest3';
+import EnterPrimaryContact from './EnterPrimaryContact';
+import close from '../../../icons/cross-out.svg';
+
+class RecurringDeliveryRequestController extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            step: 1
+        };
+    }
+
+    enterPrimaryContact(){
+        this.setState({
+            step: 4
+        });
+    }
+
+    savePrimaryContact(){
+        // TODO: save fields to DB
```

Comment:
we won't actually save anything to db until they hit the `Confirm` button in "Are you sure". So save the info they entered into `state` here for later when we write to db.